dev/integration_tests/abstract_method_smoke_test/lib/main.dart

```diff
@@ -0,0 +1,59 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_inappbrowser/flutter_inappbrowser.dart';
+
+void main() => runApp(MyApp());
+
+class MyApp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: 'Flutter Demo',
+      theme: ThemeData(
+        primarySwatch: Colors.blue,
+      ),
+      home: HomePage(),
+    );
+  }
+}
+
+class HomePage extends StatefulWidget {
+  @override
+  _HomePage createState() => _HomePage();
+}
+
+class _HomePage extends State<HomePage> {
+  @override
+  void initState() {
+    super.initState();
+
+    // Trigger the second route.
+    // https://github.com/flutter/flutter/issues/40126
+    WidgetsBinding.instance.addPostFrameCallback((_) async {
+      Navigator.of(context).push(
+          MaterialPageRoute<void>(builder: (_) => SecondPage()));
+    });
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return const Scaffold();
+  }
+}
+
+class SecondPage extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: Column(
+        children: <Widget>[
+          TextFormField(),
+          const Expanded(
+            child: InAppWebView(
```

Comment:
Tested it out and we can use `google_maps_flutter` here instead, which I think I'd prefer since that's in `flutter/plugins`.

```suggestion
            child: GoogleMap(initialCameraPosition: CameraPosition(target: LatLng(0, 0))),
```

`AndroidManifest.xml` also needs to be updated to have a property for an API key. I'd just add a sample one (`<meta-data android:name="com.google.android.geo.API_KEY" android:value="your API key"/>`) since we don't need it to render.

Reply:
sg. Done